blockchain/My-Wallet-V3-iOS
# fix(HB): IOS-1401 - Removing Min/Max buttons. Layout cleanup.

## Objective

This removes the min and max buttons. Additionally this fixes some layout issues that were present on the exchange screen. 

## Description

In addition to removing the min and max buttons, we're now accounting for the currency symbol when setting the horizontal constraint on the `primaryAmountLabel`. We're no longer hiding the `secondaryAmountLabel` beneath the `errorLabel`. The `primaryAmountLabel`, `secondaryAmountLabel` and `errorLabel` are all within the same container view. This container view is in another container that is pinned to the `TradingPairView` and the `conversionRateView`. The container view holding the `UILabels` is centered vertically. The `primaryAmountLabel` now shrinks up to 30% its original size should the user enter in a very long value like `88888.88888888 BTC`. There's now rules applied for the number of integers and fractional values that the user can input. Lastly the exchange/swap button was moved per design input. 

## How to Test

1. Build and run
2. Go to the exchange
3. Look at the exchange on various device sizes
4. Input in values, try to break the UI. 
5. The min/max buttons should be gone. 

## Screenshot/Design assessment

![exchange_ui_work](https://user-images.githubusercontent.com/41585563/46704681-b7711b00-cbf1-11e8-9505-f0a261719008.png)
![exchange_ui_work_b](https://user-images.githubusercontent.com/41585563/46704698-cce64500-cbf1-11e8-8c92-664ff8399a94.png)


## Merge Checklist
